internal/updater/providers/slickvpn/url.go

```diff
@@ -0,0 +1,57 @@
+package slickvpn
+
+import (
+	"strings"
+	"unicode"
+)
+
+func parseOpenvpnURL(url, protocol string) (country, region, city string) {
```

Comment:
Note that since we're currently using the IP info lookup (though I'd like to change this to pull data from the SlickVPN location page) - I don't think that this is even being used any more.